cmd/ocm/list/quota/cmd.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package quota
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-cli/pkg/dump"
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	amv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+var args struct {
+	json bool
+	org  string
+}
+
+var Cmd = &cobra.Command{
+	Use:   "quota",
+	Short: "Retrieve cluster quota information.",
+	Long:  "Retrieve cluster quota information of a specific organization.",
+	Args:  cobra.NoArgs,
+	RunE:  run,
+}
+
+func init() {
+	// Add flags to rootCmd:
+	flags := Cmd.Flags()
+	flags.BoolVar(
+		&args.json,
+		"json",
+		false,
+		"Returns a list of resource quota objects in JSON.",
+	)
+	flags.StringVar(
+		&args.org,
+		"org",
+		"",
+		"Specify which organization to query information from. Default to local users organization.",
+	)
+}
+
+func run(cmd *cobra.Command, argv []string) error {
+
+	// Create the client for the OCM API:
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		return fmt.Errorf("Failed to create OCM connection: %v", err)
+	}
+	defer connection.Close()
+	orgID := args.org
+
+	// Organization to search in case one was not provided:
+	if args.org == "" {
+		// Get organization of current user:
+		userConn, err := connection.AccountsMgmt().V1().CurrentAccount().Get().
+			Send()
+		if err != nil {
+			return fmt.Errorf("Can't retrieve current user information: %v", err)
+		}
+		userOrg, _ := userConn.Body().GetOrganization()
+		orgID = userOrg.ID()
+	}
+
+	// Get connection
+	orgCollection := connection.AccountsMgmt().V1().Organizations().Organization(orgID)
+	orgResponse, err := orgCollection.Get().Send()
+	if err != nil {
+		return fmt.Errorf("Can't retrieve organization information: %v", err)
+	}
+	quotaClient := orgCollection.QuotaSummary()
+
+	// Simple output:
+	if !args.json {
+
+		// Request
+		quotasListResponse, err := quotaClient.List().
+			Send()
+		if err != nil {
+			return fmt.Errorf("Failed to retrieve quota: %v", err)
+		}
+
+		// Display quota information:
+		fmt.Printf("Cluster quota for organization '%s' ID: '%s'\n",
+			orgResponse.Body().Name(), orgResponse.Body().ID())
+		quotasListResponse.Items().Each(func(quota *amv1.QuotaSummary) bool {
+			var byoc string
+			if quota.BYOC() {
+				byoc = " BYOC"
```

Comment:
The account-manager field is still named "byoc", but in output I think we want to call this "CCS".

Reply:
this is not a new command - I am just changing the command syntax. Specific enhancements will be done outside this change

Reply:
I was thinking that as soon as we add `ocm list quota` with specific behavior, somebody might start using it and changing it later may come with question of backward compatibility.
While you have it on an unmerged branch, it's easier to re-imagine it however we want.
We can still do reviews step-by-step.  For the goal of copying `ocm account quota` exactly, this LGTM :100:.

This is not a strong argument, I'm OK with merging and iterating, your call.